### ISSUE TITLE
App Crash

### ISSUE THREAD

**Issue by j33l** [CSU-Northridge-ARCS-Dev/smart-clothing-app#25]
When you click on the 3-dot menu at top right corner, the app crashes with following error message...
## Branch - Main

![image](https://github.com/CSU-Northridge-ARCS-Dev/smart-clothing-app/assets/45571080/7d654132-0560-43e9-94b4-ba9e2894cdaa)

# Error Message

![image](https://github.com/CSU-Northridge-ARCS-Dev/smart-clothing-app/assets/45571080/ea53e47a-ad78-4b3d-9318-337f77aac19b)

**Comment by j33l** [comment]
The Issue encountered after this [merge](https://github.com/CSU-Northridge-ARCS-Dev/smart-clothing-app/commit/9c5e04e49851d5c30255fb68b9d7441166f7b23f) and temporarily fixed by this [commit](https://github.com/CSU-Northridge-ARCS-Dev/smart-clothing-app/commit/b8ea1cc84a25b7f13b9ce42a788d82fc9bdab11b). @nirvanafan205

**Comment by qtdinh** [comment]
@j33l 
![Screenshot_4](https://github.com/CSU-Northridge-ARCS-Dev/smart-clothing-app/assets/59325099/7f50b1ae-9868-4ba6-b072-cc876c99ae4a)

Pretty sure I was paired up with Matthew for this task. I think I figured out the problem, you used a string for the width instead of an integer, so this was solely a styling problem. I changed it into an integer value, and it no longer crashes for me. Let me know if you have any questions.
